compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeSceneMediator.uikit.kt

```diff
@@ -73,12 +76,15 @@ internal sealed interface SceneLayout {
     class Bounds(val rect: IntRect) : SceneLayout
 }
 
+private const val FEATURE_FLAG_ACCESSIBILITY_ENABLED = false
```

Comment:
Let's combine it with other feature flags

Reply:
Can you elaborate on this remark?

Reply:
We have several independent places with feature flags and it's hard to track the status of all of them. My suggestion here is to unify our approach to do such things.

Reply:
This one is temporary, will be removed altogether when it's done. Let's adhere to KISS.